python/demos/Non_Stationary_Exploration.ipynb

```diff
@@ -0,0 +1,273 @@
nYF/w2cSxudVSXtk6Qhss9Z6AQIPUD6QEtao5EwwGXjdGNMGcFlrv6l0LI+KZzAF55PP2jwmUpPfA3MC/+cMgJ5PaWA64yRAvzHGrDDGvG+MGaznVBqKwN+RPwZeM8ZsAZbhLKHcAj2j0oCE6d/NBvW8KmmPPFWn+zdhiULOGMaYXwNdgKmBXSd6Bm0dHBMJMsb0x+ka92QNh/V8SkMRhdNDaZ21ti/OMKOXcJbb1XMqYRcYn/sr4GprbSdgMPBs4LCeUWlowvFMNpjnVUl7ZCkAkssnQTDGGJzqe35Yo5JGyxhzFzAcuNJae9hauzuwv22lZp2oeAbzccbE1eYxkaouBroBucaYPJyxb/8G+oGeT2kwtgB+4AUAa+1qIBc4F/ScSoOQCSRZa/8HwS7H24AM0DMqDUeY/v5sUM+rkvYIYq0tAj4DRgd2jQDyrLV5YQtKGi1jzB04Y3e+a63dW+nQP4BbA23OB9rhdKmreuxsnORqYaVjE40xbmNMa5xxRC9XOvYTY0yzwFjlsTgVKZFqrLUzrLVJ1tpUa20qUIgzccxb6PmUBsJauwtn8qPLAYwxnXAmNvoaPafSMJQXg7oCGGPOwRnWsR49o9Lw1PczOQ8YZoxJDBRKJxHG5zV809bLtzUReCbQZXk/ztgjkVpljEkG/gpsBhY7/1ZRYq29ALgbeN4YswEoBW4sn2cB+AvwtDFmI06F6dbApB8Az+N0aV5f3tZa+yWAtXaJMeYVYG3g2EvW2rfr9CalsdLzKQ3JJJxn7v8BPmCCtXa7MUbPqYSdtXanMWYi8Koxxo/T/fcWa+1WPaMSLsaYJ3AmkWsHLDLGHLTWnkM9//+7tXazMea3wP9wCt3vATl1duMnUL4EiYiIiIiIiIg0MOoeLyIiIiIiItJAKWkXERERERERaaCUtIuIiIiIiIg0UEraRURERERERBooJe0iIiIiIiIiDZSSdhEREREREZEGSkm7iIiIAGCMWWKMebvS9iXGmN+EMZ7bjTE/qGF/SJwiIiKNmdZpFxEREQCMMd0Bv7X2q8D274B7rLUxYYonD3jbWjupyv6QOEVERBozT7gDEBERkYbBWruuLq9vjInCSbZ9p3Oduo5TRESkIVH3eBERkTpmjHnGGPOVMWaAMeYTY8xhY8xqY8zgSm2qdfk2xrQzxlhjzJhK+/KMMTONMbcFXh8yxrxhjGljjOlgjHndGHPAGLPRGDPyFOMMxhCosv8WiA7EYI0xSyq17WqMmWeMKTbGHDHGLDbGZFa5XnmsPzfGbAaOAh2NMYnGmNmBGI8YY3KNMX8zxsRXPhfoBEys9P5jjvOzGhDYf9gYs88YM98Yc84x4hlvjNkc+Dm9Y4xJrdLuJ8aYNYFr7Q38zqp10xcREakPqrSLiIjUjwTgKeDPQBHwG2C+MSbVWrvnFK91FdAZuA04C3gEmA0kAy8DjwITgDnGmI+ttZu/Rbzl17sRuDiwbz9AIMn9ANgQeJ/DwB3AEmNMF2vtN5WuMxToBdwFlADFQAfgADAF2A2kAr8GXgcGBc4bBvwLWI7zMwPYVFOgxpgs4D3gY+AGIAaYDrxvjOltrS2q1PxKoCtwO9AUeBB4ARgYuNZFwDPAQ4E20UAG0Pp4PywREZG6oqRdRESkfrQCLrPWroJgJflLnCTyhVO8lgf4obX2aOBaPYE7gV9Yax8O7PsEJ/EdBvz1VIO11hYaYwqdl/bDKod/BxwCBltrDwXe7z2cpPpO4J5KbWOAK6y1Byrt2wf8onzDGPM/YCOwzBjT3Vq7zlr7mTGmBCiq4f2r+g3OhwHfq/QzWY7zocLPgGmV2jYBrrLWHgm0awn8zRjTwVq7FbgAKLbW3lHpnLdO8P4iIiJ1Rt3jRURE6kdRecIesB7wAR2/xbWWlienla4F8O/yHdbavTgV/W9z/RP5HrAQKDHGeIwxHqAMWAb0qyHWygk7xnG7MeYLY8zhSucCpH+LeC4CFlT+mVhrt+D0BhhUpe3S8oQ9oHx8fPnP6VOglTHmeWPMd40xzb9FPCIiIrVGSbuIiEj9KK68Ya31A16cSvSp2ltlu7Sm9wjsr4uZ39sCt+Ik25W/rgVSqrTdWcP5t+N0S38bGI5T3R4eOPZt4m0N7Khh/w6qd2uv6WcUfF9r7WLgeuCcQHy7jTGvGWOq3peIiEi9UPd4ERGRhuEoTtftyhrqOOrdOFX9x2o4VlJlu6a1Za8B3rLW3lm+wxhzyWnEswdoV8P+doFjp8Ra+xLwUqDr/JU4wwueAy759iGKiIh8O0raRUREGoYC4CpjjLvSkmiXhzMgnCp0lDHGFegZUO4dnMnZVllrvd/iuk2pXhm/8RjvfzKV9/eBHxljflFpTHtHYADw/75FfEBwiMGLxpjzgdHf9joiIiKnQ0m7iIhIw/AKMB5nUrRXgCxgbHhD4kucoXR3GGP+C+y31n4N3At8ArxrjJkJbMOZHf87wBZr7aMnuO6/gTuNMb8APgd+SM1V7C+By4wx38OpmOdaa3fX0O5+nFnm/22MeZCK2eP3UHNvgGMyxkwP3MtinA8WzsFJ2P9zKtcRERGpLRrTLiIi0gBYa/+DM6P6d4F/4iyzFu7q7uvAkzjLtX0EzAKw1ubhTDhXADyMU3l/AGeJuI9P4rq/x1lWbSrwKpCIM468ql8DeYE2n+Ak99VYaz8Dyte8fxH4P+Ar4MIqy72djI9xEvXHcBL1e3Fm9590itcRERGpFcbamoaaiYiIiIiIiEi4qdIuIiIiIiIi0kBpTLuIiEgjZ4wxgPs4TWylye9ERESkAVGlXUREpPG7mOprqlf+2hS+0EREROR4VGkXERFp/D4Fzj/O8aprq4uIiEgDoYnoRERERERERBoodY8XERERERERaaCUtIuIiIiIiIg0UEraRURERERERBooJe0iIiIiIiIiDZSSdhEREREREZEGSkm7iIiIiIiISAP1/wGftwmo1YU0fwAAAABJRU5ErkJggg==\n",
```

Comment:
I think its best not to commit the graphics, they will be auto rendered when built for the docs site

Reply:
Removed